### PR TITLE
adds program_3.py

### PR DESCRIPTION
**Pull request by shivani121794**
A new program is added which is related to str and char and demonstrates the application of `.find` method

### CODE REVIEW

File: april_revision/program_3.py
Comment (uday0097) [inline april_revision/program_3.py:1]: Good work @shivani121794 Please add comments to your code.

Comment (shivani121794) [reply april_revision/program_3.py:1]: Hey @uday0097 updated the comments

Comment (uday0097) [reply april_revision/program_3.py:1]: LGTM

**Comment by uday0097** [comment]
LGTM

**Pull request merged**